### ISSUE TITLE
Introduce new label - low priority

### ISSUE THREAD

**Issue by PeterDaveHello** [cdnjs/cdnjs#11862]
I just created a new label - [Low Priority](https://github.com/cdnjs/cdnjs/labels/Low%20Priority), which can be used to be applied on those libraries we could add but with lower priority, this kind of libraries, we can take them as assignment, practice for the intern members.

@cdnjs/intern3 please help mark the issues with this label properly in future, or if you remember any of that issues related with you, I believe that the issues were mentioned, decided by only once person currently - me, to promote some libraries with not so poopular but with good versioning, file/directory structure, documentation, etc. Please also remove their **Library - wait for notable popularity** in the meantime(I believe most of them have that label), thanks.

**Comment by PeterDaveHello** [comment]
@cdnjs/intern3 sign in here to let me know you read this, then we can close this issue.

also cc @cdnjs/active-contributor @cdnjs/library-reviewer

**Comment by sufuf3** [comment]
Got it.

**Comment by extend1994** [comment]
OK!
